packages/flutter/lib/src/cupertino/text_selection.dart

```diff
@@ -23,8 +23,8 @@ const double _kSelectionHandleRadius = 6;
 const double _kArrowScreenPadding = 26.0;
 
 /// Draws a single text selection handle with a bar and a ball.
-class _TextSelectionHandlePainter extends CustomPainter {
-  const _TextSelectionHandlePainter(this.color);
+class _CupertinoTextSelectionHandlePainter extends CustomPainter {
```

Comment:
I changed this name just so that I could uniquely refer to it in a test (there is another _TextSelectionHandlePainter in material/text_selection.dart).